kompendium-oas/src/main/kotlin/io/bkbn/kompendium/oas/schema/SimpleSchema.kt

```diff
@@ -13,5 +13,15 @@ data class SimpleSchema(
   val minLength: Int? = null,
   val maxLength: Int? = null,
   val pattern: String? = null,
-  val format: String? = null
-) : TypedSchema
+  val format: String? = null,
+  val xml: Xml? = null,
+  ) : TypedSchema {
+
+    @Serializable
+    data class Xml(
+      val name: String? = null,
+      val namespace: String? = null,
+      val prefix: String? = null,
+      val attribute: Boolean? = null,
+    )
```

Comment:
We should consolidate this into an `XmlObject` data class that can be shared across all schema types, rather than nested data classes inside each schema type.

Reply:
It's only valid on those two types and even then only wrapped is valid on Array that's why I split it out and made them separate and in their various places.

Reply:
it doesn't make sense to have it on an Object or Reference etc.

Reply:
Ahh ok I see... that makes sense now.  I'm still not sold on nesting it on a per schema basis, but we can definitely only make it available on those two schemas if that is the case

Reply:
I'm just not a fan of having the `wrapped` parameter on both since it's invalid/not used on the `simple` schema

Reply:
hence the separation